File: mint-location/src/main/java/net/crystalapps/mint/location/library/callbacks/EasyLocationCallback.java
package net.crystalapps.mint.location.library.callbacks;

import android.location.Location;

/**
 * Created by Syed Owais Ali on 10/20/2018.
 */

public interface EasyLocationCallback {
    void onResult(Location location);
    default void onFailed(String error){}
}

File: mint-location/src/main/java/net/crystalapps/mint/location/library/callbacks/EasyLocationCallback.kt
package net.crystalapps.mint.location.library.callbacks

import android.location.Location

/**
 * Created by Syed Owais Ali on 10/20/2018.
 */

interface EasyLocationCallback {
    fun onResult(location: Location)
    fun onFailed(error: String) {}
}
